addons/point_of_sale/static/src/app/screens/payment_screen/payment_screen.js

```diff
@@ -82,11 +82,15 @@ export class PaymentScreen extends Component {
     get selectedPaymentLine() {
         return this.currentOrder.get_selected_paymentline();
     }
-    async addNewPaymentLine(paymentMethod) {
+    async addNewPaymentLine(paymentMethod, automatic_payment) {
         // original function: click_paymentmethods
-        const result = this.currentOrder.add_paymentline(paymentMethod);
+        const result = this.currentOrder.add_paymentline(paymentMethod, automatic_payment);
         if (result) {
             this.numberBuffer.reset();
+            if (automatic_payment) {
+                const newPaymentLine = this.paymentLines.at(-1);
```

Comment:
```suggestion
                const newPaymentLine = this.payment_ids.at(-1);
```

Reply:
I don't understand this change because
`paymentLines = this.currentOrder.payment_ids`

Reply:
true, my bad.